test/test_fake_quant.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch.jit
+import numpy as np
+import unittest
+from common_utils import run_tests
+
+
+# Reference method for quantizing a tensor.
+def _fake_quantize_per_tensor_affine_reference(X, scale, zero_point, num_bits):
+    quant_min, quant_max = 0, 2 ** num_bits - 1
+    res = (np.clip(np.round(X / scale) + zero_point, quant_min, quant_max) - zero_point) * scale
+    res = res.reshape(X.shape)
+    return res
+
+
+# Reference method for the gradient of the quantizer.
+def _fake_quantize_per_tensor_affine_grad_reference(X, dY, scale, zero_point, num_bits):
+    Xq = np.round(X / scale) + zero_point
+    quant_min, quant_max = 0, 2 ** num_bits - 1
+    mask = np.logical_and(Xq >= quant_min, Xq <= quant_max)
+    res = dY[mask].reshape(dY.shape)
+    return res
+
+
+class TestFakeQuantizePerTensorAffine(unittest.TestCase):
+    """Tests the forward path of the FakeQuantizePerTensorAffine op."""
+    def test_forward(self):
```

Comment:
Can you add a test where a quantize_linear(), followed by a dequantize() produces consistent output for the forward?